Podfile

```diff
@@ -6,6 +6,9 @@ target 'HackerNewsReader' do
 pod 'HackerNewsKit', :path => 'Frameworks/HackerNewsKit'
 pod 'HackerNewsNetworker', :path => 'Frameworks/HackerNewsNetworker'
 
+# The SDK repo is not public yet.
+pod 'Sift', :path => '../..'
```

Comment:
This will be put in `sift-ios/samples/HackerNewsReader` directory as a git submodule. That's why it's `../..` here.